src/tools/cephfs/Dumper.cc

```diff
@@ -45,15 +45,21 @@ int Dumper::init(mds_role_t role_)
   auto fs =  fsmap->get_filesystem(role.fscid);
   assert(fs != nullptr);
 
-  JournalPointer jp(role.rank, fs->mds_map.get_metadata_pool());
-  int jp_load_result = jp.load(objecter);
-  if (jp_load_result != 0) {
-    std::cerr << "Error loading journal: " << cpp_strerror(jp_load_result) << std::endl;
-    return jp_load_result;
+  if (type == std::string("mdlog")) {
```

Comment:
I believe it's unnecessary to do create `std::string("mdlog")`, right? Just do: `type == "mdlog"`?

Reply:
yes, i think you are right, should be no need, i was doing this to